pkg/api/request.go

```diff
@@ -324,7 +324,15 @@ func (a *API) GetAccessInstructions(w http.ResponseWriter, r *http.Request, requ
 		return
 	}
 
-	apio.JSON(ctx, w, res.JSON200, http.StatusOK)
+	switch res.StatusCode() {
+	case http.StatusOK:
+		apio.JSON(ctx, w, res.JSON200, http.StatusOK)
+	case http.StatusNotFound:
+		apio.JSON(ctx, w, res.JSON404.Error, res.StatusCode())
+	case http.StatusBadRequest:
+		apio.JSON(ctx, w, res.JSON400.Error, res.StatusCode())
+	}
```

Comment:
We should have a final default error response here just to be sure

Reply:
Cheers Josh, should be covered now :)